src/main/java/com/vmware/vipclient/i18n/filters/VIPComponentFilter.java

```diff
@@ -110,7 +110,6 @@ public void init(FilterConfig filterConfig) throws ServletException {
             } catch (VIPClientInitException e) {
                 logger.error(e.getMessage());
             }
-            gc.initializeVIPService();
```

Comment:
why remove this line?

Reply:
Because the method has just been deprecated.